contracts/stellar-example/src/contract.rs

```diff
@@ -64,10 +61,7 @@ impl CustomInterchainTokenExecutable for Example {
     type Error = ExampleError;
 
     fn __interchain_token_service(env: &Env) -> Address {
-        env.storage()
-            .instance()
-            .get(&DataKey::InterchainTokenService)
-            .expect("ITS not found")
+        DataKey::get_interchain_token_service(env).expect("ITS not found")
```

Comment:
I think we use lower case for most of `.expect` messages, can you double check?
```suggestion
        DataKey::get_interchain_token_service(env).expect("interchain token service not found")
```